src/libraries/System.Net.Http/src/System/Net/Http/HttpResponseMessage.cs

```diff
@@ -177,27 +177,27 @@ public HttpResponseMessage EnsureSuccessStatusCode()
 
         public override string ToString()
         {
-            StringBuilder sb = new StringBuilder();
+            ValueStringBuilder sb = new ValueStringBuilder(stackalloc char[512]);
 
-            sb.Append("StatusCode: ");
-            sb.Append((int)_statusCode);
+            sb.Append($"StatusCode: {(int)_statusCode}");
```

Comment:
Moving multiple appends into an interpolated string is unfortunately going to result in more temporary string allocations right now since `ValueStringBuilder` doesn't have a custom handler implemented

Reply:
Fixed by using `AppendSpanFormattable()` in 4948b61.